src/colmap/estimators/CMakeLists.txt

```diff
@@ -27,6 +27,9 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+if(NOT FETCH_POSELIB)
+    find_package(PoseLib REQUIRED)
```

Comment:
This is already done in src/thirdparty/CMakeLists.txt. I guess the reason this doesn't work is because of the order in which we add sub-directories. The add_subdirectory() calls in the main CMakeLists.txt in L221 need to be swapped?

Reply:
I pushed some fixes to this branch. Please let me know if this doesn't work for you.